make_level.py

```diff
@@ -0,0 +1,217 @@
+import sys
+import os
+from shutil import copyfile
+
+cwd = os.getcwd()
+
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version
+    if not os.path.exists(os.path.join(skill_path, "__init__.py")):
+        with open(os.path.join(skill_path, "__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources",
+                "",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill",
+                "",
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")",
+                ""
+                ])
+            fp.write(level_setup)
+
+        # Read file
+        with open(os.path.join("gitgud", "skills", "__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "\n".join([
+            "from gitgud.skills.{0} import skill as {0}_skill".format(skill_name),
+            "",
+            "from gitgud.skills.u"
+        ])
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+
+        # Add to input array of Skill
```

Comment:
This comment isn't immediately clear. How about "Add skill to AllSkills"

Reply:
done